elasticsearch/prep-install.origin

```diff
@@ -3,6 +3,6 @@ if [ -z "${OSE_ES_URL:-}" ] ; then
    OSE_ES_URL=https://github.com/fabric8io/openshift-elasticsearch-plugin/releases/download/openshift-elasticsearch-plugin-${OSE_ES_VER}/openshift-elasticsearch-plugin-${OSE_ES_VER}.zip
 fi
 if [ -z "${PROMETHEUS_EXPORTER_URL:-}" ] ; then
-    PROMETHEUS_EXPORTER_URL=https://distfiles.compuscene.net/elasticsearch/elasticsearch-prometheus-exporter-${PROMETHEUS_EXPORTER_VER}.zip
+    PROMETHEUS_EXPORTER_URL=https://github.com/lukas-vlcek/elasticsearch-prometheus-exporter/releases/download/${PROMETHEUS_EXPORTER_VER}/elasticsearch-prometheus-exporter-${PROMETHEUS_EXPORTER_VER}.zip
```

Comment:
Is it necessary to define the default value here again when it has been already defined in `elasticsearch/Dockerfile.centos7` (line 56)?

Reply:
Presumably, the `ARG PROMETHEUS_EXPORTER_URL` would apply here, so the condtion `-z "${PROMETHEUS_EXPORTER_URL:-}"` should be false.

Reply:
Do you think we can remove whole `if [ -z "${PROMETHEUS_EXPORTER_URL:-}" ] ; then ... fi` statement?

Reply:
You could try removing it and see what happens . . .